src/UndockedRegFreeWinRT/mwinrtact/mwinrtact.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Runtime.InteropServices;
+
+namespace Microsoft.Windows
+{
+    public static class UndockedRegFreeWinrt
+    {
+        [DllImport("winrtact.dll")]
```

Comment:
Do we want a failed HRESULT to throw in this case? If so, we need PreserveSig=false on the DllImport
https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.dllimportattribute.preservesig?view=netcore-3.1

Reply:
I think if something goes wrong with initializing the detours and loading the catalog. We want to silently fail?

Reply:
@BenJKuhn What do you think?

Reply:
The app has to opt into this functionality, and if the manifests don't load, the app is likely to break in strange ways. So I think an exception here are reasonable and appropriate.